3-semantic-html/semantic.html

```diff
@@ -0,0 +1,69 @@
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial=scale=1.0">
+        <meta name="description" content="Jaime Moreno Curriculum Vitae">
+        <meta name="keywords" content="CV, curriculum vitae, learning, job search">
+        <meta name="author" content="Jaime Moreno"> 
+        <title>WORLD PANDEMIC</title>
+    </head>
+
+    <body>
+        <header>
+            <h1>
+                CONTAGIOUS DISEASES
+            </h1>
+            <nav>
+                <ul>
+                    <li><a href="https://www.youtube.com/watch?v=CXOsVXybVJY"> New Coronavirus</a></li>
+                    <li><a href="https://www.youtube.com/watch?v=Vg4N0wUeUWo"> Smallpox</a></li>
+                    <li><a href="https://www.youtube.com/watch?v=Qw2OBCfXsEo"> Blackdeath</a></li>
+                </ul>
+            </nav>
+        </header>
+        
+        <main>
+            <article>
+                <header>
+                    <h2>CORONAVIRUS</h2>
+                </header>
+                <section>
+                    <blockquote>
+                        The 2019 novel coronavirus (2019-nCoV) is a new strain of coronavirus that first appeared in Wuhan, China, in December 2019. Though it was only just discovered, 2019-nCoV has already spread rapidly in China and around the world. As of Feb. 10, 2020, the virus has led to more than 40,000 illnesses and 900 deaths in China, as well as more than 400 illnesses and two deaths outside of mainland China. (The vast majority of cases and deaths have occurred in Hubei Province, where Wuhan is located.)
+                        <br>
+                        - from
+                        <cite>
+                            28 Devastating Infectious Diseases
+                        </cite>
+                        by <a href="https://www.livescience.com/13694-devastating-infectious-diseases-smallpox-plague.html"> LiveScience</a>
+                    </blockquote>
+                </section>
+                <footer>
+                    <a href="https://www.worldometers.info/coronavirus/"> Coronavirus Stats</a>
+                </footer>
+            </article>
+        </main>
+
+        <aside>
+            <p class="aligncenter">
+                <img height="150" width="300" src="https://www.cdc.gov/coronavirus/2019-ncov/images/2019-coronavirus.png" alt="Coronavirus">
+            </p>
+            <style>.aligncenter{text-align:right;}</style>
+        </aside>
+
+        <footer>
+            <address>
+            <p>Made by: wRattlehead316</p>
+                <nav>
+                    <p>You can contact the author here:</p>
+                        <ol>
+                            <li><a href="https://www.linkedin.com/in/jaimemoreno97/"> LinkedIn</a></li>
+                            <li><a href="https://www.facebook.com/jaimeandres97"> Facebook</a></li>
+
+                    </ol>
+                </nav>
+            </address>
+        </footer>
```

Comment:
Please separate the `<address>` of the `<nav>`. You can have both of them inside the `<footer>`, but it's better if you separate them. Something like this: 

```suggestion
<footer>
             <p>You can contact the author here:</p>
            <address>
                <a href="mailto:pushdev.cuentas@gmail.com">pushdev.cuentas@gmail.com</a><br>
                <a href="tel:+13115552368">(311) 555-2368</a>
            </address>
            <p>Content related:</p>
            <nav>
                    <!-- your list items -->
            </nav>
 </footer>
```